kd_sweep.sh

```
@@ -142,6 +143,99 @@ elif [[ "$MODE" == "score_weights" ]]; then
+
+  # 13) SampledKD LinUCB k=25
+  sbatch --wait train.slurm linucb 25 light "$KD_SWEEP_TAG"
+
+  echo "[run_all_10m] All jobs submitted and completed in sequence."
+else
+  usage; exit 1
+fi
+  
+
+
+elif [[ "$MODE" == "run_sampledkd_10m" ]]; then
```

Shellcheck (0.11.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)